src/backend/marsha/core/migrations/0051_migrate_live_state.py

```diff
@@ -0,0 +1,23 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("core", "0050_alter_video_join_mode"),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql=(
+                "UPDATE video "
+                "SET live_state=upload_state, upload_state='pending' "
+                "WHERE upload_state='harvesting' OR upload_state='harvested';"
+            ),
+            reverse_sql=(
+                "UPDATE video "
+                "SET upload_state=live_state, live_state='stopped' "
+                "WHERE live_state='harvesting' OR live_state='harvested';"
+            ),
```

Comment:
Maybe I'm missing something, but I think the reverse won't work as no video will satisfy the where condition.

Reply:
You are probably missing something yes. In the new workflow `harvested` and `harvesting` belong to `live_state`. If we want to rollback to a version where they were managed by the `upload_state` we have to migrate them from `live_state` to `upload_state`. 
You should give a try you will see what is happening